controllers/utils/init.go

```diff
@@ -34,6 +35,7 @@ func RegisterSchemes(s *runtime.Scheme) {
 	utilruntime.Must(maistrav1.SchemeBuilder.AddToScheme(s))
 	utilruntime.Must(knservingv1.AddToScheme(s))
 	utilruntime.Must(authorinov1beta2.SchemeBuilder.AddToScheme(s))
+	utilruntime.Must(istioclientv1beta1.AddToScheme(s))
```

Comment:
wouldn't this break setups that does not have istio?

Reply:
No I don't think so because another istio scheme already added there. This is the second istio scheme

Reply:
can we do a test without service mesh installed?
cc @lugi0

Reply:
for unit test, yes we can test it without service mesh installed. The CRD is for unit test